templates/todo/task_list.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>TODO list</h1>
+    <button style="float: right; bottom: 50px; position: relative" class="btn btn-primary"
```

Comment:
Remove styles here and add this to styles.css:

.btn-add-task{
float: right; 
bottom: 50px; 
position: relative
}

```suggestion
    <button class="btn btn-primary btn-add-task"
```